CHANGELOG.md

```diff
@@ -1,3 +1,24 @@
+# dbt_fivetran_log v1.11.0
+[PR #141](https://github.com/fivetran/dbt_fivetran_log/pull/141) includes the following updates:
+
+## Schema Changes: Adding the transformation_runs table
+- We have added the `transformation_runs` source table. Note that not all customers have the `transformation_runs` source table, particularly if they are not using Fivetran Transformations. Therefore, the `transformation_runs` table will only populate if the table exists in your schema, via a new variable `fivetran_platform_using_transformations`, which automatically checks for the table. If the table doesn't exist, the staging `stg_fivetran_platform__transformation_runs` model will persist as an empty model and respective downstream fields will be null. 
```

Comment:
Can you add a note somewhere in the CHANGELOG about being able to override this behavior by setting `fivetran_platform_using_transformations` to False, similar to what you have in the DECISIONLOG

Reply:
added